### PR TITLE
Cut 0.2.0

### PR DESCRIPTION
**Pull request by thieman**
@jsingle @paetling

### CODE REVIEW

**Comment by paetling** [comment]
:ghost: :cow:

File: dusty/constants.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import os
 
-VERSION = '0.1.6'
+VERSION = '0.2.0'
```
Comment (jsingle) [inline dusty/constants.py]: After you cut the release, this needs to go to 2.1

**Comment by jsingle** [comment]
:guitar: :camel:

**Comment by thieman** [comment]
:love_hotel:

**Pull request merged**